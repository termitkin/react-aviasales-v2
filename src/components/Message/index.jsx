import React from "react";
import styled from "styled-components";

const StyledMessage = styled.div`
  display: block;
  margin-top: 12px;
  text-align: center;
  color: #333;
`;

const Message = ({ text }) => {
  return <StyledMessage>{text}</StyledMessage>;
};

export default Message;
