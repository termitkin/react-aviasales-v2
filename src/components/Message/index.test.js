import React from "react";
import { create, act } from "react-test-renderer";
import Message from "./index";

describe("Message component", () => {
  let wrapper;

  afterAll(() => {
    wrapper.unmount();
  });

  act(() => {
    wrapper = create(<Message text={"Some message"} />);
  });

  it("Matches the snapshot", () => {
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
