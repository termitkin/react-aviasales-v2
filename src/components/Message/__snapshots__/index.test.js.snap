// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message component Matches the snapshot 1`] = `
.emotion-0 {
  display: block;
  margin-top: 12px;
  text-align: center;
  color: #333;
}

<div
  className="emotion-0 emotion-1"
>
  Some message
</div>
`;
